changes/61a5ed1d20140ec69aea3b7931355441.yaml

```diff
@@ -0,0 +1,5 @@
+---
+desc: Update the allowed versionf of the ``packaging`` and ``xxhash`` libraries.
```

Comment:
```suggestion
desc: Update the allowed version of the ``packaging`` and ``xxhash`` libraries.
```

Reply:
it only lets me apply the change once. @MichaelSquires won the race :laughing: :racing_car:

Reply:
:racehorse: